arcSDinstall.bat

```diff
@@ -0,0 +1,19 @@
+winget install docker.dockerdesktop
+@echo Please open docker desktop and config the proxies.

```

Comment:
Move this line after the `reboot` hint?

Also, I think we need a `@pause` here to wait for the user to launch the docker desktop (to make sure docker engine is up). Or can we open docker desktop for users from this script?